### ISSUE TITLE
Add more maintainers

### ISSUE THREAD

**Issue by thoughtless** [joshbuddy/anybase#6]
Would you be willing to add @bfad and @mdiebolt as maintainers? If you'd prefer, I'm sure that @polleverywhere would be willing to adopt this repo.

I'm happy to continue being a maintainer as well, but since I'm no longer actively working on a project using this gem, I wanted to invites some others as maintainers.

NOTE: I also invited them as owners through Rubygems.org.

**Comment by bfad** [comment]
Could I be added as a maintainer to this repo?
@joshbuddy would you be OK with this gem being adopted by [Poll Everywhere](https://github.com/polleverywhere/) to maintain?

**Comment by joshbuddy** [comment]
Yeah that sounds fine. I’ll add you in the morning On 15 Sep 2022, at 22.04, Brad Lindsay ***@***.***> wrote:﻿
Could I be added as a maintainer to this repo?
@joshbuddy would you be OK with this gem being adopted by Poll Everywhere to maintain?

**Comment by bfad** [comment]
Was your response, "Yeah that sounds fine" in response to making me a maintainer on this repo, or to having Poll Everywhere adopt the gem (or both)? If you're good with Poll Everywhere taking over ownership of the gem, then you don't need to make me an owner of this repo, I can just create a new one under the "polleverywhere" organization and update the gem to point to it.

**Comment by joshbuddy** [comment]
I meant both. Want to email me your rubygems email address?

**Comment by joshbuddy** [comment]
Actually, you're already on the rubygems one. I've added you to the repo as well.

**Comment by bfad** [comment]
@joshbuddy and @thoughtless 

I moved the repo to [the Poll Everywhere organization](https://github.com/polleverywhere/anybase). In the process, I also updated it to require MFA to publish to Ruby gems. If you are still interested in being a maintainer, please let me know and setup MFA with rubygems.org. I'll add you to the new repo and add you back to the rubygems ownership.

**Comment by joshbuddy** [comment]
Hey brad. Thanks for doing all thisI’ll add mfa right away and ping you back On 3 Oct 2022, at 16.38, Brad Lindsay ***@***.***> wrote:﻿
@joshbuddy and @thoughtless
I moved the repo to the Poll Everywhere organization. In the process, I also updated it to require MFA to publish to Ruby gems. If you are still interested in being a maintainer, please let me know and setup MFA with rubygems.org. I'll add you to the new repo and add you back to the rubygems ownership.
